website/docs/r/emr_cluster.html.markdown

```diff
@@ -325,6 +325,7 @@ The following arguments are supported:
 * `security_configuration` - (Optional) The security configuration name to attach to the EMR cluster. Only valid for EMR clusters with `release_label` 4.8.0 or greater
 * `core_instance_group` - (Optional) Configuration block to use an [Instance Group](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-instance-group-configuration.html#emr-plan-instance-groups) for the [core node type](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-master-core-task-nodes.html#emr-plan-core).
 * `core_instance_fleet` - (Optional) Configuration block to use an [Instance Fleet](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-instance-fleet.html) for the core node type. Cannot be specified if any `core_instance_group` configuration blocks are set. Detailed below.
+* `log_encryption_kms_key_id` - (Optional) The AWS KMS master key ID used for encrypting log files. This attribute is only available with EMR version 5.30.0 and later, excluding EMR 6.0.0.  
```

Comment:
As per https://docs.aws.amazon.com/emr/latest/APIReference/API_JobFlowDetail.html,
this is Customer Master Key (CMK). Could you please replace `master key` with `customer master key (CMK)` to align with AWS documentation?

Reply:
Thanks, Updated